src/main/java/com/alkemy/ong/model/Category.java

```diff
@@ -0,0 +1,41 @@
+package com.alkemy.ong.model;
+
+import lombok.*;
+import org.hibernate.annotations.CreationTimestamp;
+import org.hibernate.annotations.SQLDelete;
+import org.hibernate.annotations.UpdateTimestamp;
+import org.hibernate.annotations.Where;
+
+import javax.persistence.*;
+import java.sql.Timestamp;
+
+@Entity
+@Table(name = "categories")
+@Getter
+@Setter
+@ToString
+@AllArgsConstructor
+@NoArgsConstructor
+@SQLDelete(sql = "UPDATE categories SET deleted = true WHERE id = ?")
+@Where(clause = "deleted = false")
+public class Category {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "category_id")
+    private Long id;
+    @NonNull
```

Comment:
Agregale el mensaje de error al @NotNull, ejemplo: `@NotNull(message = "the name cannot be empty")`